ChaosMod/Effects/db/Player/PlayerRandomHud.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+	Effect By OnlyRealNubs
+*/
+
+#include "stdafx.h"
+
+static int selectedComponent = 0;
+
+static std::vector<int> availableComponents = { 1, 2, 3, 6, 7, 8, 9, 10, 14, 15, 16, 19, 20, 22 };
+
+static void OnStart()
+{
+	selectedComponent = comps[g_Random.GetRandomInt(0, availableComponents.size() - 1)];

```

Comment:
Forgot to rename „comps“ here

Reply:
[9275920](https://github.com/gta-chaos-mod/ChaosModV/pull/2849/commits/9275920e84c693f90f786636e50693eccec79b01)

Reply:
👍 (although you miswrote my name in your commit 😏)

Reply:
> 👍 (although you miswrote my name in your commit 😏)

I spelt it *en francais* 🥖

Reply:
Oh la la